get-started/index.md

```diff
@@ -177,22 +177,54 @@ Once you have authenticated through the cluster's SSO login you will be
 redirected back to the Cryostat web application. The redirect URL contains
 an access token for Cryostat's service account with the permissions you have
 granted to it. This access token will eventually expire and you will be
-required to log back in on the cluster SSO login page.
+required to log back in on the cluster SSO login page. The Cryostat
```

Comment:
Okie just checking my knowledge here:

I think OpenShift SSO involves some Identity Provider (IdP) like RH SSO. Does cluster username and password counts as SSO too?

Reply:
Not sure exactly what you're asking. The cluster's SSO could be something like RH SSO or some other system, so long as it implements OAuth then it should work since that's what Cryostat is targeting with its token redirection flow. If the cluster has its own user authz with cluster-specific credentials then that's what will be presented to the user, but in any case Cryostat just sees this as an OAuth flow.

Reply:
Oh nvmind I was confused (can't even rmb why I asked this question :))) Thanks for explaining.